editor/src/components/editor/store/editor-state.ts

```diff
@@ -388,16 +388,23 @@ export type EditorStoreFull = EditorStoreShared & {
   patchedDerived: DerivedState
 }
 
+type StoreName = 'editor-store' | 'canvas-store' | 'low-priority-store'
```

Comment:
After this PR (to prevent blocking it) please can we rename these to better clarify the differences between the `editor-store` and `canvas-store`

Reply:
yep! absolutely